src/admin/resolvers/mutations/ApprovedItem.integration.ts

```diff
@@ -608,6 +516,73 @@ describe('mutations: ApprovedItem', () => {
     });
   });
 
+  describe('updateApprovedCorpusAuthorsItem mutation', () => {
+    let item: ApprovedItem;
+    let authors: ApprovedItemAuthor[];
+    let input: UpdateApprovedItemAuthorsInput;
+
+    beforeEach(async () => {
+      item = await createApprovedItemHelper(db, {
+        title: "3 Things Everyone Knows About LEGO That You Don't",
+        status: CuratedStatus.RECOMMENDATION,
+        language: 'EN',
+      });
+
+      authors = [
+        { name: 'Author One', sortOrder: 1 },
+        { name: 'Author Two', sortOrder: 2 },
+      ];
+
+      input = {
+        externalId: item.externalId,
+        authors,
+      };
+    });
+
+    it('should succeed if user has full access', async () => {
```

Comment:
nit - test name could probably be better (as this isn't an auth test)?

Reply:
True! Should have updated test names when splitting them up.